### DISCUSSION TITLE
Consider adding Cloudflare Turnstile captcha  into Freeform?

### DISCUSSION THREAD

**Discussion by terryupton** [solspace/craft-freeform#1551] [Feature Requests]
Would you consider adding the ability to use a Cloudflare Turnstile captcha  into Freeform?
https://www.cloudflare.com/en-gb/products/turnstile/ 

There is a free Craft plugin that does this, but I don't think it woul integrate with Freeform. Would be nice to have this as a native captcha integration. Especially with Googles stance change of pricing and usage of their re-Captcha mechanism. 
https://plugins.craftcms.com/turnstile

Thanks
Terry

**Comment by kjmartens** [comment]
Thanks for your feedback @terryupton. I will add this to our list for future consideration. We're always looking for ways to improve Freeform's spam protection. 🙂

**Comment by kjmartens** [comment]
Support for Cloudflare Turnstile is now available in Freeform 5.6.5+ 🙂

**Comment by terryupton** [comment]
Fantastic. Thanks @kjmartens 
Will this get added to the docs?

**Reply by kjmartens** [reply]
Yes, you can see the docs for it here: 🙂

https://docs.solspace.com/craft/freeform/v5/integrations/turnstile

**Reply by terryupton** [reply]
Fantastic. Apologies. I did a quick search for 'Turnstile' and couldn't see it. Thanks again.

**Reply by kjmartens** [reply]
Yeah it probably hasn't been indexed through Algolia yet since it's so fresh 😅
